entrypoint/versionbump/flannelbump_test.go

```diff
@@ -0,0 +1,76 @@
+//go:build flannel
+
+package versionbump
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	. "github.com/rancher/distros-test-framework/pkg/template"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+)
+
+var _ = Describe("Flannel Version bump:", func() {
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(cluster)
+	})
+
+	It("Validate Node", func() {
+		testcase.TestNodeStatus(
+			cluster,
+			assert.NodeAssertReadyStatus(),
+			nil)
+	})
+
+	It("Validate Pod", func() {
+		testcase.TestPodStatus(
+			cluster,
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus())
+	})
+
+	cmd := "kubectl get node -o yaml : | grep 'hardened-flannel' -A1"
+	if cluster.Config.Product == "k3s" {
+		cmd = "/var/lib/rancher/k3s/data/current/bin/flannel"
+	}
```

Comment:
Can we put this in the `It` block below where it's used so that it is scoped only to where it is used?

Reply:
Updated